site/pages/api/publish.test.ts

```diff
@@ -174,4 +194,42 @@ describe("publish", () => {
             Location: `${REVIEW_DISRUPTION_PAGE_PATH}/${defaultDisruptionId}`,
         });
     });
+
+    it("should send an email to org admin if an org staff creates a disruption", async () => {
+        getDisruptionSpy.mockResolvedValue(disruptionWithConsequences);
+        getSessionSpy.mockImplementation(() => ({ ...mockSession, isOrgStaff: true, isSystemAdmin: false }));
+        getAllUsersInGroupSpy.mockResolvedValue(mockOrgAdmins);
+        const { req, res } = getMockRequestAndResponse({
+            body: {
+                disruptionId: defaultDisruptionId,
+            },
+            mockWriteHeadFn: writeHeadMock,
+        });
+
+        await publish(req, res);
+
+        expect(sesMock.send.calledOnce).toBeTruthy();
+        expect(sesMock.commandCalls(SendEmailCommand)[0].args[0].input.Destination).toEqual({
+            ToAddresses: ["emailtoshow@test.com"],
+        });
+    });
+
+    it("should redirect to dashboard and log an error if an org staff creates a disruption an no org admins exists for staff member's org", async () => {
+        getDisruptionSpy.mockResolvedValue(disruptionWithConsequences);
+        getSessionSpy.mockImplementation(() => ({ ...mockSession, isOrgStaff: true, isSystemAdmin: false }));
+        getAllUsersInGroupSpy.mockResolvedValue([]);
```

Comment:
Should we be explicit in returning a list without admins and verifying that the function `getOrgAdminEmailsForStaffUserOrg` was invoked using a spy?

Reply:
This is now outdated, similar tests have been created for the ```sendDisruptionApprovalEmail``` function.

In this case, I don't think it's necessary to check if ```getOrgAdminEmailsForStaffUserOrg``` has been invoked since we mock the required inputs and are checking for the correct outputs. Although I have created unit tests for this function to ensure given certain inputs we get the expected outputs.